### ISSUE TITLE
Locations.ini is overwritten when upgrading

### ISSUE THREAD

**Issue by HolgerPollyNet** [NOA-ReACT/PollyXT-SCC-Pipelines#4]
While doing upgrade via pip, the locations.ini is overwritten as well. This should be changed in my opinion.

**Comment by thgeorgiou** [comment]
Hello! Are you putting new locations at the files described [here](https://noa-react.github.io/PollyXT-SCC-Pipelines/locations.html#adding-new-locations)? If so, it would be very unusual (not to mention dangerous) that the app goes and deletes user files.

If convenient (& relatively stable), you could send me configurations to add them to the stock `locations.ini`.

**Comment by HolgerPollyNet** [comment]
Indeed, my fault, I did not add it to the .config folder because it was for some reason not working, so I edited the "main" locations.ini...sorry.

This are my additional entries:

[Kuopio]
scc_code = kuo
lat = 62.89238
lon = 27.67703
altitude_asl = 0.1
daytime_configuration = 522
nighttime_configuration = 523
channel_id = 1525, 1526, 1527, 1528, 1529, 1530, 1531, 1532, 1533, 1534, 1535,	1536
background_low = 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
background_high = 249, 249, 249, 249, 249, 249, 249, 249, 249, 249, 249, 249
lr_input = 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1
temperature = 15
pressure = 1000
total_channel_355_nm = 0
cross_channel_355_nm = 1
total_channel_532_nm = 4
cross_channel_532_nm = 5
calibration_355nm_channel_ids = 1554, 1556, 1553, 1555
calibration_532nm_channel_ids = 1544, 1546, 1543, 1545
profile_name = KUOPIO
sounding_provider = noa_wrf

[Cyprus]
scc_code = cyc
lat = 35.095192
lon = 33.203430
altitude_asl = 0.1
daytime_configuration = 578
nighttime_configuration = 579
channel_id = 1617, 1618, 1619, 1620, 1621, 1622, 1623, 1624, 1625, 1626, 1627, 1628, 1633
background_low = 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
background_high = 240, 240, 240, 240, 240, 240, 240, 240, 240, 240, 240, 240, 240
lr_input = 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,1
temperature = 15
pressure = 1000
total_channel_355_nm = 0
cross_channel_355_nm = 1
total_channel_532_nm = 4
cross_channel_532_nm = 5
calibration_355nm_channel_ids = 1609, 1610, 1613, 1615
calibration_532nm_channel_ids = 1611, 1612, 1614, 1616
profile_name = CYPRUS
sounding_provider = noa_wrf

[Arielle]
scc_code = lei
lat = 51.343479
lon = 12.387772
altitude_asl = 125
daytime_configuration = 580
nighttime_configuration = 581
channel_id = 601,  602, 603,  604, 605, 606, 607, 608, 609, 610, 611, 612, 1631
background_low = 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
background_high = 240, 240, 240, 240, 240, 240, 240, 240, 240, 240, 240, 240, 240
lr_input = 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1
temperature = 15
pressure = 1000
total_channel_355_nm = 0
cross_channel_355_nm = 1
total_channel_532_nm = 4
cross_channel_532_nm = 5
calibration_355nm_channel_ids = 613, 614, 617, 619
calibration_532nm_channel_ids = 615, 616, 618, 620
profile_name = ARIELLE
sounding_provider = noa_wrf

[TROPOS_lei]
scc_code = tro
lat = 51.343479
lon = 12.387772
altitude_asl = 125
daytime_configuration = 548
nighttime_configuration = 512
channel_id = 1388, 1389, 1390,  1391, 1392, 1393, 1394, 1395, 1396, 1397, 1398, 1399, 1697
background_low = 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
background_high = 240, 240, 240, 240, 240, 240, 240, 240, 240, 240, 240, 240, 240
lr_input = 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1
temperature = 15
pressure = 1000
total_channel_355_nm = 0
cross_channel_355_nm = 1
total_channel_532_nm = 4
cross_channel_532_nm = 5
calibration_355nm_channel_ids = 1400, 1401, 1404, 1406
calibration_532nm_channel_ids = 1402, 1403, 1405, 1407
profile_name = TROPOS
sounding_provider = noa_wrf

[Lacros]
scc_code = pal
lat = -53.1548
lon = -70.9113
altitude_asl = 0.1
daytime_configuration = 457
nighttime_configuration = 458
channel_id = 1125,1126,1127,1128,1129,1130,1131,1132,1133,1134,1135,1136,1632
background_low = 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
background_high = 240, 240, 240, 240, 240, 240, 240, 240, 240, 240, 240, 240, 240
lr_input = 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1
temperature = 15
pressure = 1000
total_channel_355_nm = 0
cross_channel_355_nm = 1
total_channel_532_nm = 4
cross_channel_532_nm = 5
calibration_355nm_channel_ids = 1137, 1138, 1141, 1143
calibration_532nm_channel_ids = 1139, 1140, 1142, 1144
profile_name = LACROS
sounding_provider = noa_wrf

**Comment by thgeorgiou** [comment]
Hello @HolgerPollyNet,

I've implemented most calibration-related functionality, could you add the calibration system IDs to these locations so I can include them in the next version?

Specifically, I added `calibration_configuration_355nm` and `calibration_configuration_532nm` for the two wavelengths.

**Comment by HolgerPollyNet** [comment]
I will do so and send you a email with the new locations.ini. One question, for our systems the calibration configuration is the same for 355 and 532. So I assume I enter the same number twice?

**Comment by thgeorgiou** [comment]
@HolgerPollyNet Yep the same number twice is OK!

Thanks
